package.json

```diff
@@ -1,31 +1,45 @@
 {
   "author": "Felix Geisendörfer <felix@debuggable.com> (http://debuggable.com/)",
-  "name": "measured",
-  "description": "This is an alternative port of Coda Hale's metrics library.",
```

Comment:
I think it'd be great to keep a reference somewhere to the original project. Maybe somewhere in the README?

Reply:
So it still is at the top of the readme.

![image](https://user-images.githubusercontent.com/711726/40123504-3ffe26aa-58db-11e8-9b77-ea91b3aafb93.png)

I did tweak the language before because it used to reference Coda Hale's GoLang port of the original java library which he also wrote, with other people too. 

It is a bit confusing because @codahale is a person but some people refer to him as the library, and the Dropwizard Metrics library is the original as far as I can tell. Also the Dropwizar Metrics lib is much more feature complete and advanced.

Reply:
Let me know if you want to re-word that or add to it?